.github/workflows/sbom.yml

```diff
@@ -26,11 +26,15 @@ jobs:
           python -m pip install --upgrade setuptools
           python -m pip install --upgrade wheel
           python -m pip install --upgrade sbom4python
+          python -m pip install --upgrade sbom2doc
           pip install . --upgrade --upgrade-strategy=eager
       - name: Generate SBOM for cve-bin-tool
         run: |
           sbom4python --module cve-bin-tool --output cve-bin-tool-py${{ matrix.python }}.spdx
           sbom4python --module cve-bin-tool --sbom cyclonedx --format json --output cve-bin-tool-py${{ matrix.python }}.json
+      - name: Generate Markdown for SBOM
+        run: |
+          sbom2doc --input-file cve-bin-tool-py${{ matrix.python }}.spdx --format markdown --output-file cve-bin-tool-py${{ matrix.python }}.html
```

Comment:
This appears to generate a markdown version of the sbom but then it doesn't look like you use it anywhere. What was the goal here?

Reply:
@terriko The idea I had was to create a release note which included the SBOM information. I suggested that we use my sbom2doc utility as it produces a markdown version of the SBOM and the idea (which I probably didn't share with @Rexbeast2) was to use this file as part of a bigger document. Does this seem like a resonable goal @terriko ?

Reply:
Sounds good long-term, but if that's how it's going to be used, let's add it into a docs build step rather than updating it every week alongside the SBOM.  So we can take it out of here and build it into a different PR when we're actually prepared to use it.  They can be generated at any time since we'll have the weekly sboms anyhow.